exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillResultSetImpl.java

```diff
@@ -1384,7 +1402,7 @@ public void updateRowId( String columnLabel, RowId x ) throws SQLException {
 
   @Override
   public int getHoldability() throws SQLException {
-    throwIfClosed();
+    throwIfTimedOutOrClosed();
```

Comment:
Removed the timeout check for most getter methods, exceptions being the getResultSet() related APIs.